processor/groupbyattrsprocessor/processor_test.go

```diff
@@ -869,6 +869,55 @@ func TestCompacting(t *testing.T) {
 	}
 }
 
+func Test_GetMetricInInstrumentationLibrary(t *testing.T) {
+	m := pmetric.NewMetric()
+	m.SetName("metric")
+	m.SetDescription("description")
+	m.SetUnit("unit")
+	d := m.SetEmptyGauge().DataPoints().AppendEmpty()
+	d.SetDoubleValue(1.0)
+
+	m2 := pmetric.NewMetric()
```

Comment:
Maybe a comment here, highlighting the difference between `m` and `m2` i.e with and without datapoints, since this if this test.